cmake/Findzstd.cmake

```diff
@@ -0,0 +1,53 @@
+#######################
```

Comment:
From a quick look at the zstd GitHub page, it looks like at least the most recent version has a CMake config that I would hope gets included in the packages for each distro. Is it still necessary to write our own find script?